<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>741</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Fournisseur</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>621</width>
        <height>361</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter Fournisseur</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>351</width>
          <height>321</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Produit</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Matricule fiscale</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_nom">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>40</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_matricule">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>70</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_produit">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>100</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Adresse</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_adresse">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>130</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>180</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Num√©ro</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_numero">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>170</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Rib</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>250</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Date d'ajout</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_rib">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>200</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>280</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_date">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>250</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Afficher Fournisseur</string>
       </attribute>
       <widget class="QTableView" name="tabfournisseur">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>571</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Supprimer Fournisseur</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>85</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Matricule</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_matricule_supp">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Modifier Fournisseur</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Commande Achat</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>621</width>
        <height>361</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Ajouter Commande</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-50</y>
          <width>601</width>
          <height>381</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_11">
         <attribute name="title">
          <string>ajouter commande</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>20</y>
            <width>491</width>
            <height>351</height>
           </rect>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>141</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Num√©ro de commande</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>56</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Prix</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>101</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Fournisseur</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>100</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>70</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>40</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>101</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Quantit√©</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_23">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>160</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_24">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>230</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>240</y>
             <width>161</width>
             <height>81</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_7">
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>160</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_5">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>130</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_6">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>160</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Produit</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_7">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>190</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_12">
         <attribute name="title">
          <string>Afficher commande</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>10</y>
            <width>491</width>
            <height>351</height>
           </rect>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_25">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>56</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Produit</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_26">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>56</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Prix</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_27">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>101</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Fournisseur</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_21">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>100</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_22">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>70</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_23">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>40</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_28">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>101</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Quantit√©</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_29">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>160</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
          <widget class="QDateEdit" name="dateEdit_2">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>160</y>
             <width>110</width>
             <height>22</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_6">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>130</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="spinBox_2">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>130</y>
             <width>42</width>
             <height>22</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_30">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>220</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_24">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>220</y>
             <width>161</width>
             <height>81</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_5">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>170</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_8">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>210</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Rechercher</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Afficher Commande</string>
       </attribute>
       <widget class="QTableView" name="tabfournisseur_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>571</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Modifier Commande</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Supprimer Commande</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>85</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Num√©ro de commande</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_matricule_supp_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
